CONTRIBUTING.md

```diff
@@ -16,18 +26,33 @@ Please read the agreement and acknowledge it by ticking the appropriate box in t
 
 - [x] Tick to sign-off your agreement to the Developer Certificate of Origin (DCO) 1.1
 
+<!-- Append library specific information here
```

Comment:
Remove this since everything below is commented and hidden in markdown view.

Reply:
Fixed in 0f19672b217943ef931b6274e724df1612662c7b